G/GoogleTest/build_tarballs.jl

```diff
@@ -3,22 +3,28 @@
 using BinaryBuilder, Pkg
 
 name = "GoogleTest"
-version = v"1.10.0"
+version = v"1.11.0"
 
-# Collection of sources required to complete build
-sources = [
-    GitSource("https://github.com/google/googletest.git", "703bd9caab50b139428cea1aaff9974ebee5742e")
+ # Collection of sources required to complete build
+ sources = [
+     GitSource("https://github.com/google/googletest.git", "e2239ee6043f73722e7aa812a459f54a28552929")
 ]
 
 # Bash recipe for building across all platforms
 script = raw"""
+
+if [[ "${target}" == *apple* ]]; then
+    CMAKE_FLAGS+=(-DCMAKE_CXX_FLAGS=-mmacosx-version-min=11.0)
+fi
```

Comment:
This isn't necessary for aarch64-apple-darwin: https://github.com/JuliaPackaging/BinaryBuilderBase.jl/pull/193.  And it looks rather optimistic for x86_64